example/Example.jsx

```diff
@@ -77,6 +77,10 @@ export default class Example extends React.PureComponent<Props, State> {
           rangeSelect
           firstMonth={subMonths(new Date(), 1)}
           lastMonth={addMonths(new Date(), 1)}
+          customClasses={{
+            class1: [new Date()],
+            class2: [new Date(), addDays(new Date(), 2)],
```

Comment:
add also class1 and class2 with some style somewhere just for example

Reply:
ok